datasets/makhzan/makhzan.py

```diff
@@ -0,0 +1,120 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""An Urdu text corpus for machine learning, natural language processing and linguistic analysis."""
+
+from __future__ import absolute_import, division, print_function
+
+import logging
+import xml.etree.ElementTree as ET
+from pathlib import Path
+
+import datasets
+
+
+_CITATION = """\
+@InProceedings{huggingface:dataset,
+title = {A great new dataset},
+authors={huggingface, Inc.
+},
+year={2020}
+}
+"""
+
+_DESCRIPTION = """\
+An Urdu text corpus for machine learning, natural language processing and linguistic analysis.
+"""
+
+_HOMEPAGE = "https://matnsaz.net/en/makhzan"
+
+_LICENSE = "All files in the /text directory are covered under standard copyright. Each piece of text has been included in this repository with explicity permission of respective copyright holders, who are identified in the <meta> tag for each file. You are free to use this text for analysis, research and development, but you are not allowed to redistribute or republish this text. Some cases where a less restrictive license could apply to files in the /text directory are presented below. In some cases copyright free text has been digitally reproduced through the hard work of our collaborators. In such cases we have credited the appropriate people where possible in a notes field in the file's metadata, and we strongly encourage you to contact them before redistributing this text in any form. Where a separate license is provided along with the text, we have provided corresponding data in the publication field in a file's metadata."
+
+_DOWNLOAD_URL = "https://github.com/zeerakahmed/makhzan/archive/master.zip"
```

Comment:
Could you specify the latest commit sha in the url ?
This way if the repo's content changes in the future this script will keep working.

For example you can use this url instead (it uses the latest commit sha):
```
https://github.com/zeerakahmed/makhzan/archive/99db56552d6781dcd184bdd3466bce15fd0a1ec0.zip
```

Reply:
Perfect. I reduced the dummy data to 2 files and fixed SHA of repo.